src/components/ButtonRow.tsx

```diff
@@ -0,0 +1,37 @@
+import React from "react";
+import styled from "@emotion/styled";
+import { CardDropdownButton } from "components/CardDropdownButton";
+
+interface Props {
+  containerRef: React.MutableRefObject<HTMLDivElement>;
+  rowButtons: JSX.Element[];
+  cardItems: JSX.Element[];
+  cardLoading: boolean;
+  cardDisabled: boolean;
+  setIsVisibleCard: (v: boolean) => void;
+  isVisibleCard: boolean;
+}
+
+export const ButtonRow = (props: Props) => {
+  return (
+    <Container ref={props.containerRef}>
+      {props.rowButtons}
+      <div>
+        <CardDropdownButton
+          disabled={props.cardDisabled}
+          cardItems={props.cardItems}
+          isVisibleCard={props.isVisibleCard}
+          setIsVisibleCard={props.setIsVisibleCard}
+          loading={props.cardLoading}
```

Comment:
very in favor of destructuring the props instead of using `props.` for every single one. doing it this way is inconsistent with all other component in the codebase.